### DISCUSSION TITLE
TP5 - Uso de la expresion ternaria

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#154] [Preguntas y respuestas] [answered]
Hola! estaba buscando maneras de cambiar el valor de una variable sin tener que recurrir a los ifs y me encontre con las expresiones ternarias. Podemos usarlas para el tp? mando el ejemplo de lo que queria implementar en el codigo:

`char jugador_actual = (turno_jugador == 0) ? 'X' : 'O';`

**Comment by martinvilu** [comment, marked as answer]
Nope, la legibilidad sufre sin una ganancia clara.

**Reply by martinvilu** [reply]
Una función es equivalente y más flexible
